### ISSUE TITLE
feature request: delete unused images

### ISSUE THREAD

**Issue by mkduew** [trendschau/typemill#456]
please see my video: https://youtu.be/ETHfNSr26os

**Comment by trendschau** [comment]
yes, that is on the list already, but prio is a bit lower. I will probably plan a release with improvements of the media library end of the year

**Comment by trendschau** [comment]
As of version 2.17.0 there i a filter button in the media library that filters images that are not linked in content pages,, meta files, and user data. This is no 100% guaranty that the image is not used anywhere else, but it can help you to clean up the media library.
